### ISSUE TITLE
Command aliases cause duplicates

### ISSUE THREAD

**Issue by alpha0010** [laravel-zero/laravel-zero#310]
Continuing from https://github.com/nunomaduro/laravel-console-summary/issues/9.

If I use aliases
```php
<?php
namespace App\Console\Commands;

use LaravelZero\Framework\Commands\Command;

class Projects extends Command
{
    protected $signature = 'projects';
    protected $description = 'List all running projects';

    public function __construct()
    {
        parent::__construct();
        $this->setAliases(['p']);
    }
// ...
```

I get
```
$ ./application

  Application  unreleased

  USAGE: application <command> [options] [arguments]

  projects     List all running projects
  projects     List all running projects

  app:build    Build a single file executable
  app:install  Install optional components
  app:rename   Set the application name

  make:command Create a new command
```

**Comment by patrickdaze** [comment]
Looking into this one…

**Comment by owenvoke** [comment]
This should now be resolved by updating Laravel Console Summary to v1.6.0, but let us know if you have any further issues. 👍🏻
